src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,70 @@ redirects:
   </figcaption>
-</LandingPageHero>
-
-<LandingPageTileGrid>
-  <LandingPageTile
-    title="Learn about alerting in New Relic."
-    href="/docs/alerts/new-relic-alerts/getting-started/alerting-new-relic"
-    icon="fe-zap"
-  >
-     Understand the fundamental principles of policies, conditions, thresholds, notification channels, and events.
-  </LandingPageTile>
-
-{' '}
-<LandingPageTile
-  title="Follow alert policy workflow."
-  href="/docs/alerts/new-relic-alerts/configuring-alert-policies/alert-policy-workflow"
-  icon="fe-move-horizontal"
->
-  Learn the basic process for creating an alert policy for New Relic products.
-</LandingPageTile>
-
-{' '}
-<LandingPageTile
-  title="Define alert conditions."
-  href="/docs/alerts/new-relic-alerts/configuring-alert-policies/define-alert-conditions"
-  icon="fe-sliders"
->
-  Set the necessary criteria for New Relic Alerts to create an incident. Or, use
-  the [REST API](/docs/alerts/new-relic-alerts/rest-api-alerts).
-</LandingPageTile>
-
-{' '}
-<LandingPageTile
-  title="Create notification channels."
-  href="/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts"
-  icon="fe-message-circle"
->
-  Specify how to get notified when critical violations occur, so you can quickly
-  fix the issue.
-</LandingPageTile>
-
-  <LandingPageTile
-    title="Learn about AI."
-    href="/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence"
-    icon="fe-alert-triangle"
-  >
-
-    Take advantage of automatically enabled [Proactive Detection](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-new-relic-ai) to be warned about trouble brewing in your APM-monitored apps. Group similar incidents with [Incident intelligence](/docs/alerts-applied-intelligence/incident-intelligence/get-started-incident-intelligence). [Incident workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/enhance-notifications-incident-workflows) enrich your notifications with New Relic data.
-
-  </LandingPageTile>
-
-  <LandingPageTile
-    title="Resolve an issue."
-    href="/docs/alerts/new-relic-alerts/reviewing-alert-incidents"
-    icon="fe-tool"
-  >
-     [Acknowledge an incident](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/acknowledge-alert-incidents) after it occurs, or [close a violation manually](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/close-violations-manually) as needed.
-  </LandingPageTile>
-</LandingPageTileGrid>
+
+In the New Relic UI, Alerts & AI is divided into several sections, each unlocking different features. Check out the section explanations below to learn how these features can meet your needs.
+
+## **Analyze** your issues, incidents, and anomalies [#analyze]
+
+Use the analyze features to scan for critical problems affecting your systems, via alerts violations, issues, incidents, and anomalies.
```

Comment:
Here again we're referring to violations alongside incidents, as if they're different things. It seems to me that this will confuse matters even more than they already are confused (given that we're using the same term to mean something different).

Maybe we should define the terminology change right at the top of this doc?

Reply:
Of course, the Overview page's charts all show "violations," which also adds to the confusion. I really don't know what the best solution is here (besides not using "incidents" as a new term, but that ship has long sailed).

Reply:
I think not mentioning things specifically present in the UI isn't the way to address this.

I think the terminology change would be a great thing to mention in the Transition guide doc.

Reply:
I'm going to leave violations here for now until the UI changes, anyway.